cypress/support/step-definitions/carousel-steps.js

```diff
@@ -82,3 +100,7 @@ When('I set transition to {string}', (transition) => {
 Then('transition is set to {string} with {string}', (transition, direction) => {
   giveTransition(transition, direction).should('exist');
 });
+
+When('I click clickable slide {int}', () => {
```

Comment:
```suggestion
When('I click clickable slide', () => {
```

Reply:
Removed slide index from the test scenario and this step 👍